opm/input/eclipse/Schedule/ScheduleGrid.cpp

```diff
@@ -17,6 +17,7 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include "opm/input/eclipse/Schedule/Schedule.hpp"
 #include <opm/input/eclipse/Schedule/ScheduleGrid.hpp>
```

Comment:
Please keep the CPP file's own header (i.e., `ScheduleGrid.hpp` in this case) as the first include statement.  This prevents certain kinds of issues from getting into the code base.